test/fixtures/ignore-preprocessor-args/parseFile-out.graph

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
the diagram id could now be part of the output in the diagram node. It is nothing I need, it just came to my mind 😉